environment.yml

```diff
@@ -22,6 +22,7 @@ dependencies:
   - ucimlrepo=0.0.7
   - vegafusion=1.6.9
   - vegafusion-python-embed=1.6.9
+  - quarto=1.5.57
```

Comment:
```suggestion
```
Remove this change or untrack this file to avoid a merge conflict as `environment.yml` will be updated in a different pull request.